common/pageinfo/src/main/java/com/quran/data/page/provider/madani/MadaniDataSource.kt

```diff
@@ -2,7 +2,7 @@ package com.quran.data.page.provider.madani
 
 import com.quran.data.source.QuranDataSource
 
-internal open class MadaniDataSource : QuranDataSource {
+open class MadaniDataSource : QuranDataSource {
```

Comment:
can you undo this change? since you're making a new one in the test and not mocking it, it should work.

Reply:
Changed, the reason I changed it at first place is to avoid this inspection warning
"Usage of kotlin internal declaration from different module"

The test still works after reverting back although.